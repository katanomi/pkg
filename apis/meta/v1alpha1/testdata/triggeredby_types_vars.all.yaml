user:
  kind: "ServiceAccount"
  apiGroup: "v1"
  name: "default"
  namespace: "default"
cloudEvent:
  id: 5aa832f6-17eb-11ed-9832-1bc8c2fab4b0
  source: https://github.com/katanomi/spec
  specversion: "1.0"
  subject: dc442de0cb4efac54104bc247d387a13432edad8
  time: "2022-08-09T13:58:36Z"
  type: dev.katanomi.cloudevents.github.push
  datacontenttype: application/json
  extensions:
    action: create
    branch: main
    commit: dc442de0cb4efac54104bc247d387a13432edad8
    commitmessage: 'fix: buildrun label description (#137)  Co-authored-by: jzli <jzli@alauda.io>'
    knativearrivaltime: "2022-08-09T13:58:36.60007265Z"
    reference: refs/heads/main
    repository: katanomi/spec
ref:
  kind: "SomeKind"
  namespace: "default"
  name: "some-object"
  uid: "41ce460c-cd91-493f-a59c-8f2ccdea9034"
  apiVersion: somegroup/v1alpha1
triggeredTimestamp: "2022-08-09T13:58:36Z"
triggeredType: Automated
