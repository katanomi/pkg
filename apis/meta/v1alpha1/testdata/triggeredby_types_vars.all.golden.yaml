triggeredBy.user: ""
triggeredBy.user.kind: "ServiceAccount"
triggeredBy.user.apiGroup: "v1"
triggeredBy.user.name: "default"
triggeredBy.user.namespace: "default"
triggeredBy.cloudEvent: ""
triggeredBy.cloudEvent.id: 5aa832f6-17eb-11ed-9832-1bc8c2fab4b0
triggeredBy.cloudEvent.source: https://github.com/katanomi/spec
triggeredBy.cloudEvent.specversion: "1.0"
triggeredBy.cloudEvent.subject: dc442de0cb4efac54104bc247d387a13432edad8
triggeredBy.cloudEvent.time: "2022-08-09T13:58:36Z"
triggeredBy.cloudEvent.type: dev.katanomi.cloudevents.github.push
triggeredBy.cloudEvent.datacontenttype: application/json
triggeredBy.cloudEvent.extensions: ""
triggeredBy.cloudEvent.extensions.action: create
triggeredBy.cloudEvent.extensions.branch: main
triggeredBy.cloudEvent.extensions.commit: dc442de0cb4efac54104bc247d387a13432edad8
triggeredBy.cloudEvent.extensions.commitmessage: 'fix: buildrun label description (#137)  Co-authored-by: jzli <jzli@alauda.io>'
triggeredBy.cloudEvent.extensions.knativearrivaltime: "2022-08-09T13:58:36.60007265Z"
triggeredBy.cloudEvent.extensions.reference: refs/heads/main
triggeredBy.cloudEvent.extensions.repository: katanomi/spec
triggeredBy.ref: ""
triggeredBy.ref.kind: "SomeKind"
triggeredBy.ref.namespace: "default"
triggeredBy.ref.name: "some-object"
triggeredBy.ref.uid: "41ce460c-cd91-493f-a59c-8f2ccdea9034"
triggeredBy.ref.apiVersion: somegroup/v1alpha1
triggeredBy.triggeredTimestamp: "2022-08-09T13:58:36Z"
triggeredBy.triggeredType: Automated
